idea-plugin/src/main/kotlin/arrow/meta/ide/phases/integration/integration.kt

```diff
@@ -1,14 +1,56 @@
 package arrow.meta.ide.phases.integration
 
 import arrow.meta.phases.ExtensionPhase
-import arrow.meta.phases.resolve.PackageProvider
-import arrow.meta.phases.resolve.synthetics.SyntheticResolver
 import com.intellij.openapi.project.Project
 
 interface SyntheticResolver : ExtensionPhase {
-  fun syntheticResolver(project: Project): SyntheticResolver?
+  fun syntheticResolver(project: Project): arrow.meta.phases.resolve.synthetics.SyntheticResolver?
 }
 
 interface PackageProvider : ExtensionPhase {
-  fun packageFragmentProvider(project: Project): PackageProvider?
```

Comment:
I think Kotlin can resolve that for instance, `SyntheticResolver` is meant to be `arrow.meta.phases.resolve.synthetics.SyntheticResolver`. 
But if it didn't work that's fine here, too. 

:D looks good

Reply:
If I understand correctly, it will look better, but it may not be at all obvious that a different type is returned here, and not the one that was just declared.